helix-term/src/commands.rs

```diff
@@ -1922,6 +1923,15 @@ enum Extend {
 }
 
 fn extend_line(cx: &mut Context) {
+    let (view, doc) = current_ref!(cx.editor);
+    let extend = match doc.selection(view.id).primary().direction() {
+        Direction::Forward => Extend::Below,
+        Direction::Backward => Extend::Above,
+    };
```

Comment:
This would be surprising for me.

Reply:
I use extend_above a lot but there is no enough key space to bind it so that I was thing we can provide both functions in one command.
This is not bound to `x` as default, can we keep it so that we can rebind it to this function if need?

Reply:
I think this is actually very clever. I'm def going to rebind x to this myself.

Reply:
I tried it out again, doesn't seemed like an issue to me.